docs/customizing-the-library/creating-custom-fields-and-widgets.md

```diff
@@ -8,7 +10,7 @@
 
 ### How us-forms-system uses rjsf
 
-The us-forms-system code uses rjsf to render form fields, but it builds a scaffolding on top of it to support multi-page forms and our common form patterns. Additionally, us-forms-system uses rjsf to create a form configuration spec that allows devs to specify the structure of one of our multi-page forms.
+The us-forms-system code uses rjsf to render form fields, but it builds a scaffolding on top of it to support multi-page forms and our common form patterns. Additionally, us-forms-system uses rjsf to create a form configuration spec that allows developers to specify the structure of a multi-page form.
```

Comment:
The "our common form patterns" supposes a particular use case so I'd pull out "our".